WinUIGallery/ContentIncludes.props

```diff
@@ -360,9 +361,11 @@
     <Content Include="ControlPagesSampleCode\CommandBarFlyout\CommandBarFlyoutSample1_xaml.txt" />
     <Content Include="ControlPagesSampleCode\XamlUICommand\XamlUICommandSample1_cs.txt" />
     <Content Include="ControlPagesSampleCode\XamlUICommand\XamlUICommandSample1_xaml.txt" />
-    <Content Include="ControlPagesSampleCode\Window\CreateWindowSample1.txt" />
-    <Content Include="ControlPagesSampleCode\Window\Titlebar\TitlebarSample1.txt" />
     <Content Include="ControlPagesSampleCode\Window\Titlebar\TitlebarSample2.txt" />
+    <Content Include="ControlPagesSampleCode\Window\Titlebar\TitleBarSample1_cs.txt" />
```

Comment:
We renamed every instance of "Titlebar" to "TitleBar". This might create a conflict when we merge.